juriscraper/AbstractSite.py

```diff
@@ -25,7 +26,9 @@
     phantomjs_executable_path = _legacy_path
     msg = """Please place phantomjs executable in /usr/local/bin.
     See https://github.com/freelawproject/juriscraper/pull/241"""
-    raise DeprecationWarning(msg)
+    # oops don't raise DeprecationWarning(msg)
```

Comment:
I don't think we need these comments here. Can you remove them please before we pull?

Reply:
done.